src/flag_gems/runtime/error/error.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+
+def PASS(e):
+    pass
+
+
+class ErrorHandler:
+    def __init__(self, callback_fn=None):
+        logging.basicConfig(
+            level=logging.ERROR, format="%(asctime)s - %(levelname)s - %(message)s"
+        )
```

Comment:
Consider using a custom logger for flaggems and do not use the root logger, since it affects other libraries that uses logging.